### ISSUE TITLE
Automatic Let's Encrypt certificate renewal failing on python3-acme 2.1.0

### ISSUE THREAD

**Issue by DanWin** [globaleaks/GlobaLeaks#3571]
### What version of GlobaLeaks are you using?

Latest globaleaks version (4.12.8) on Debian bookworm

### What browser(s) are you seeing the problem on?

All

### What operating system(s) are you seeing the problem on?

Linux

### Describe the issue

Since Updating the repo to use the bookworm packages, we see the following error in our logs and we get an email, that renewal has failed:
```
2023-07-25 03:26:15+0000 [-] [E] [1] Automatic HTTPS renewal failed ('NoneType' object has no attribute 'update')
2023-07-26 02:33:11+0000 [-] [E] [1] Automatic HTTPS renewal failed ('NoneType' object has no attribute 'update')
2023-07-27 01:25:11+0000 [-] [E] [1] Automatic HTTPS renewal failed ('NoneType' object has no attribute 'update')
2023-07-28 01:26:35+0000 [-] [E] [1] Automatic HTTPS renewal failed ('NoneType' object has no attribute 'update')
2023-07-29 02:40:52+0000 [-] [E] [1] Automatic HTTPS renewal failed ('NoneType' object has no attribute 'update')
2023-07-30 03:18:16+0000 [-] [E] [1] Automatic HTTPS renewal failed ('NoneType' object has no attribute 'update')
2023-07-31 03:25:40+0000 [-] [E] [1] Automatic HTTPS renewal failed ('NoneType' object has no attribute 'update')
2023-08-01 02:30:22+0000 [-] [E] [1] Automatic HTTPS renewal failed ('NoneType' object has no attribute 'update')
2023-08-02 01:26:23+0000 [-] [E] [1] Automatic HTTPS renewal failed ('NoneType' object has no attribute 'update')
2023-08-03 02:35:04+0000 [-] [E] [1] Automatic HTTPS renewal failed ('NoneType' object has no attribute 'update')
 ```

**Comment by evilaliv3** [comment]
Than you for reporting this @DanWin 

This was never reported before, if you can just wait let's see if it is a transient error that will be resolved at the next attempt or not.

**Comment by DanWin** [comment]
@evilaliv3 This issue persists daily since updating to bookworm last week. Unless it was fixed in yesterdays new release, it will likely re-occure.

**Comment by evilaliv3** [comment]
I see Dan, thank you for pointing this out.

I will look if python3-acme have significan changes in the usage of the API that requires update in globaleaks.

Eventually if i identify a possibly patch would you be able to test it before release?

We will also spin up a bookworm instance to retest live.

**Comment by evilaliv3** [comment]
The version of python3-acme is effectively changed a lot passing from version 1.12 to 2.1.0

**Comment by DanWin** [comment]
Yes, of course. I'm happy to test a patch prior to release. I've also changed error logging to debug level now, to catch any other possibly useful information during renewal.

**Comment by evilaliv3** [comment]
Thank you @DanWin

**Comment by evilaliv3** [comment]
I think it could be related to changes in recent versions of python acme that are included in bookworm: https://github.com/certbot/certbot/pull/9307

**Comment by evilaliv3** [comment]
@alexzorin @osirisinferi: i suspect there could be some breaking change between 1.12 and 2.1.0 that has broken the following code pattern widely used in many softwares.

Are you already aware of the change that should be applied? Thank you for your attention

```
    client = create_v2_client(directory_url, accnt_key)

    try:
        client.net.account = client.new_account(
            messages.NewRegistration.from_data(
                terms_of_service_agreed=True
            )
        )

    except errors.ConflictError as error:
        existing_reg = messages.RegistrationResource(uri=error.location)
        existing_reg = client.query_registration(existing_reg)
        client.update_registration(existing_reg)
```

Basically the call to client.query_registration seems to trigger the exception mentioned in the ticket opening.
The problem is probably that the re-regenerated RegistrationResource does not have any "body" attribute so the call existing_reg.body.update() results in a failure.

**Comment by evilaliv3** [comment]
@DanWin : fixed! the fix will be included in release 2.12.9 by today

**Comment by DanWin** [comment]
Thanks @evilaliv3 , confirmed it is working :)
